### PR TITLE
Refactor: optimize form meta migration query

### PR DESCRIPTION
**Pull request by pauloiankoski**
## Description

While fixing migration bugs, I noticed that the serialized form meta data was being re-serialized during migration, causing fatal errors due to changes in the data structure.

Initially, I considered unserializing the data before inserting it into the database and then serializing it again. However, after discussing it, we decided to completely revamp the way we migrate form metas.

In this Pull Request, we have replaced the previous approach of using a for-loop to store all data in memory and then performing individual insertions. Instead, we now use a single `INSERT INTO` query to streamline the data insertion process, improving both the accuracy and speed of the migration.

## Affects

Form Migration

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205692499999969

### CODE REVIEW

File: src/FormMigration/Steps/FormMeta.php [hunk 1]
```diff
@@ -17,13 +18,20 @@ public function process()
...
+                INSERT INTO $formMetaTable (form_id, meta_key, meta_value)
+                SELECT
+                    $newFormId,
+                    meta_key,
+                    meta_value
+                FROM
+                    $formMetaTable
+                WHERE
+                    form_id = $oldFormId
+                "
```
Comment (JasonTheAdams) [inline src/FormMigration/Steps/FormMeta.php:34]: Let's be good kids and use `DB::prepare` to insert the `$newFormId` and $oldFormId`. 🍭

**Review by JasonTheAdams — APPROVED** [review]
Love this improvement, @pauloiankoski! This will be more reliable and _way_ faster! One minor suggestion, but I'll approve.

File: src/FormMigration/Steps/FormMeta.php (hunk 1 above)
Comment (pauloiankoski) [reply src/FormMigration/Steps/FormMeta.php]: Added here: ddb4fb3e86a412f9c079df0d6bc6bd086d42babb

**Review by rickalday — APPROVED** [review]
Passed manual QA tests

**Pull request merged**